core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -447,13 +448,11 @@ abstract class AbstractControllerBrokerRequestBatch(config: KafkaConfig,
 
     leaderAndIsrRequestMap.filterKeys(controllerContext.liveOrShuttingDownBrokerIds.contains).foreach {
       case (broker, leaderAndIsrPartitionStates) =>
-        if (stateChangeLog.isTraceEnabled) {
-          leaderAndIsrPartitionStates.foreach { case (topicPartition, state) =>
-            val typeOfRequest =
-              if (broker == state.leader) "become-leader"
-              else "become-follower"
-            stateChangeLog.trace(s"Sending $typeOfRequest LeaderAndIsr request $state to broker $broker for partition $topicPartition")
-          }
+        leaderAndIsrPartitionStates.foreach { case (topicPartition, state) =>
+          val typeOfRequest =
+            if (broker == state.leader) "become-leader"
+            else "become-follower"
+          stateChangeLog.info(s"Sending $typeOfRequest LeaderAndIsr request $state to broker $broker for partition $topicPartition")
```

Comment:
Shouldn't this one be `debug` at least?

Reply:
I'm fine with reducing INFO level logs. My preference would be to group these and then log an info at the end `sent $typeOfRequest for partitions X`. WDYT we have both the debug and the aggregated log?